src/cli/main.ts

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env node
+
+import * as program from 'commander';
+import * as path from 'path';
+import { createNgPackage, NgPackagrCliArguments } from './../lib/ng-packagr';
+
+function parseProjectPath(parsed: string): string {
+  return parsed || path.resolve(process.cwd(), 'ng-package.json');
+}
+
+program
+  .name('ng-packagr')
+  .option('-p, --project <path>', 'Path to the \'ng-package.json\' or \'package.json\' file.', parseProjectPath)
```

Comment:
The default value must come as the 4th argument to `.option(one, two, three, default)`.

Example:

```js
const DEFAULT_PROJECT_PATH = path.resolve(process.cwd(), 'ng-package.json');

program
    .name('ng-packagr')
    .option('-p, --project <path>', 'Path to the \'ng-package.json\' or \'package.json\' file.', parseProjectPath, DEFAULT_PROJECT_PATH)
    .parse(process.argv);
```

Change above changed the default handling for calling `ng-packagr` w/o arguments, thus causing existing builds to fail (e.g. `"scripts": { "build": "ng-packagr" }`

Reply:
Fixed as requested